oidc-controller/api/routers/oidc.py

```diff
@@ -134,19 +135,16 @@ async def post_token(request: Request, db: Database = Depends(get_db)):
     ver_config = await VerificationConfigCRUD(db).get(auth_session.ver_config_id)
     presentation = client.get_presentation_request(auth_session.pres_exch_id)
     claims = Token.get_claims(presentation, auth_session, ver_config)
-    token = Token(
-        issuer="placeholder", audiences=["keycloak"], lifetime=10000, claims=claims
-    )
 
-    # modify sub to use vc-attribute as configured
-    new_sub = token.claims.pop("sub")
+    # modify subject identifier value to use vc-attribute as configured
+    new_sub = claims.pop("sub")
     provider.provider.authz_state.authorization_codes[model.get("code")][
-        "sub"
+        "public"
```

Comment:
@Jsyro this is where it was failing. It was not finding `sub` and therefore was throwing an exception. Had to review the library code to find it 🤷🏻

Reply:
hmmm, not sure how that changed, but glad you resolved it.